src/backend/distributed/master/master_repair_shards.c

```diff
@@ -297,7 +296,8 @@ CopyShardPlacement(int64 shardId, char *sourceNodeName, int32 sourceNodePort,
 
 	HOLD_INTERRUPTS();
 
-	SendCommandListInSingleTransaction(targetNodeName, targetNodePort, ddlCommandList);
+	SendCommandListToWorkerInSingleTransaction(targetNodeName, targetNodePort, nodeUser,
```

Comment:
Hmz, @byucesoy @onderkalaci  this seems wrong, this should not be done as the Citus extension owner. Otherwise, the user would not be able to access the table.